File: LeetCode/problems/3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var missingMultiple = function(nums, k) {
    let mySet = new Set([...nums]);
    const A = k;
    while(mySet.has(k)){
        k += A;
    }
    return k;
};

File: LeetCode/problems/3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.c
int missingMultiple(int* nums, int numsSize, int k) {
    bool mySet[101] = {0};
    for(int I = 0; I<numsSize; I++){
        mySet[nums[I]] = 1;
    }
    int A = k;
    while(A<101){
        if(mySet[A] == 0){return A;}
        A += k;
    }
    return A;
}
